streaming/flink/install.sh

```diff
@@ -10,6 +10,7 @@ targets=(
   "module.karpenter"
   "module.ebs_csi_driver_irsa"
   "module.eks_blueprints_kubernetes_addons"
```

Comment:
this line needs to be removed. Look for any other entries in the PR with `eks_blueprints_kubernetes_addons `

Reply:
Fixed in all repos